src/test/java/com/github/awwkoala/selenium/TestBase.java

```diff
@@ -0,0 +1,22 @@
+package com.github.awwkoala.selenium;
+
+import com.github.awwkoala.selenium.util.WebDriverProvider;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public abstract class TestBase {
+
+  protected WebDriver driver;
+
+  @BeforeClass
+  public void getDriver() {
+    driver = new WebDriverProvider().getChromeDriver();
```

Comment:
To be precise currently your `webDriverProvider's` `getChromDriver` implementation returns new ChromeDriver every time so I would call it `createChromeDriver`. Later you can think about some optimization and not creating chromeDriver every time, but providing webDriver in more 'singletonish' way.